projects/plugins/jetpack/changelog/update-new-plan-open-in-modal

```diff
@@ -0,0 +1,4 @@
+Significance: minor
+Type: enhancement
+
+Open add new subscription modal when there's hash in URL
```

Comment:
This changelog entry is misleading though, this change puts the hash in the url, not opens a new subscription modal.